app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+<!-- Modal -->
+<div id="table_<%= table.id %>_tier" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">This is a tier <%= table.scenario.tier %> scenario.</h4>
+      </div>
+      <div class="modal-body">
+        <p>You must have a character that is within the levels of <%= table.scenario.tier %> to play.<br>
+          If this is an interactive special, you can play a character on subtier higher or lower, as they are ticketed
+          by sub-tier instead of full tier, though we recommend strongly remaining within your sub-tier.
+        </p>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
+</div>
+
 <% if table.scenario.pregen_only? %>
-    <span class="badge badge-pregen">Pregen</span>
+    <!-- Trigger the modal with a button -->
+    <button type="button" class="btn btn-round-xs btn-pregen btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_pregen">Pregen</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_pregen" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Pregenerated Characters Only</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may only be played with pregenerated characters. You may not play this with your own
+              character.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+
 <% end %>
 <% if table.scenario.evergreen? %>
-    <span class="badge badge-evergreen">Evergreen</span>
+    <button type="button" class="btn btn-round-xs btn-evergreen btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_evergreen">Evergreen</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_evergreen" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Evergreen Scenario</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may be replayed on multiple characters for credit. See the Pathfinder Society Roleplaying
+              Guild Guide (or similar document for Starfinder, etc)</p>
+          </div>
```

Comment:
Reword: "You may replay this scenario with different characters. See the relevant Society RPG Guide for details."